### PR TITLE
Add Tekken 7 & 8

### PR DESCRIPTION
**Pull request by hahayupgit**
Unplayable :-(

### CODE REVIEW

File: src/game-support/tekken7.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Tekken 7
+<!-- script:Aliases [] -->
+
+{{#template ../templates/rating.md status=Garbage installs=Yes opens=No}}
+
+Tekken 7 is currently **unplayable** in Whisky.
```
Comment (mattmaniak) [inline src/game-support/tekken7.md:6]: Why is it exactly inplayable?

**Review by mattmaniak — CHANGES_REQUESTED** [review]

File: src/game-support/tekken7.md (hunk 1 above)
Comment (hahayupgit) [reply src/game-support/tekken7.md:6]: Clicking "play game" does nothing

**Comment by hahayupgit** [comment]
hahaha i was confused at your comment at first, i thought you were directly asking me for an explanation rather than asking me to add those details to the page

**Comment by mattmaniak** [comment]
Not as deep 🙃

**Review by mattmaniak — APPROVED** [review]

**Pull request merged**